Kg applications intro (#9)
greenelab/knowledge-graph-review: content/04.network_applications.md

## Applying Knowledge Graphs to Biomedical Challenges

> Knowledge graphs can be used to aid researchers in executing important biomedical tasks such as finding new treatments for FDA approved drugs [@doi:10.7554/eLife.26726] and predicting protein function in human tissues [@doi:10.1093/bioinformatics/btx252].

Reviewer: ```suggestion
Knowledge graphs can help researchers tackle many important biomedical tasks such as finding new treatments for FDA approved drugs [@doi:10.7554/eLife.26726] and predicting protein function in human tissues [@doi:10.1093/bioinformatics/btx252].
```

> Typically, the first step of these tasks involve modeling a knowledge graph's structural information (e.g., type of edges between nodes and how connected nodes are to each other).

Reviewer: Interestingly rephetio doesn't do this.
